via/via_system.cpp

```diff
@@ -2052,11 +2051,6 @@ ViaSystem::ViaResults ViaSystem::GenerateTestInfo(void) {
             full_cmd = cube_exe;
             full_cmd += " --c 100 --suppress_popups";
             int test_result = RunTestInDirectory(path, cube_exe, full_cmd);
-            if (test_result == 0) {
```

Comment:
Removing this will make it attempt both times to run cube, why not just re-add the if (test_result != 0) { continue; }?

Reply:
If the test failed, we want to output some indication of failure. The code in the loop after the continue does that.

Also, there's a PrintEndTable that is always executed after the loop is exited. If we do a continue here, the corresponding PrintBeginTable is not executed.